### ISSUE TITLE
Governance

### ISSUE THREAD

**Issue by ffmancera** [nmstate/nmstate#732]
With the increase of users and contributors to Nmstate and its cross teams membership nature, it has become obvious that documenting the decision making and the different roles in the project is needed.

Ref: https://github.com/nmstate/nmstate/pull/540

**Comment by cathay4t** [comment]
I don't want to be BDFL. Prefer voting and reasoning.

I did a quick view on GOVERNANCE.md, that sounds like text book
with mix of 'code of conduct', 'open source processing' and 'github
roles'. Seems better fit to big projects with multiple companies
contributing.

For the code of conduct, nmstate is a small project ~~dominated~~ supported by
Red Hat, we don't need information like [Linux kernel code of conduct][1].
If still required, we can just say, contributor must follow the
[Linux foundation code of conduct][2]

For the open source processing, I prefer practical HOWTO document like:

 * Who should you seek help from?
 * How to create your first PR?
 * How to debug issue?
 * How to set development environment?

For the roles, we are should be maintainers(from my point of view) with all
mighty rights. We maintain this project driven by money and love of
open source. No need to document who to do what and why.
Choice should never be made base on ranking.
We all have reputation to maintain which refrain us doing bad things.
In the small projects I contributed in, I give maintainer/full rights
to new contributors with limited vetting and nothing ever went wrong.


[1]: https://www.kernel.org/doc/html/latest/process/code-of-conduct.html
[2]: https://www.linuxfoundation.org/code-of-conduct/
